glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -562,16 +563,42 @@ open class GleanInternalAPI internal constructor () {
      * Handle the foreground event and send the appropriate pings.
      */
     internal fun handleForegroundEvent() {
-        Pings.baseline.submit(Pings.baselineReasonCodes.foreground)
         GleanValidation.foregroundCount.add(1)
```

Comment:
This slightly changes counting.
Before we submitted the ping first, then counted foreground.

Now it's the other way around. Feels like a bug fix, so just asking if that was intentionally.

Reply:
Whoops, no, not intentional. Let me roll it back.